### DISCUSSION TITLE
Fetch specific culture for item by ID using Delivery API

### DISCUSSION THREAD

**Discussion by bjarnef** [umbraco/Umbraco-CMS#17834] [Features and ideas]
Using Delivery API we can fetch a specific item by ID.
https://docs.umbraco.com/umbraco-cms/13.latest/reference/content-delivery-api#gets-a-content-item-by-id

E.g.

```
/umbraco/delivery/api/v2/content/item/d97e7d85-29e9-44ec-a376-2310e5d55c20
```

When node vary but culture and we have `da` and `en` culture, it seems to fetch the first culture.

Not sure there's a specific reason we can do this?

```
/umbraco/delivery/api/v2/content/item/d97e7d85-29e9-44ec-a376-2310e5d55c20/en
```

We do have another request e.g. fetching a settings node by `contentType` and specifically `/en` version:
https://docs.umbraco.com/umbraco-cms/13.latest/reference/content-delivery-api#gets-content-item-s-from-a-query

**Comment by jdpnielsen** [comment]
Did you try setting the [Accept-Language](https://docs.umbraco.com/umbraco-cms/13.latest/reference/content-delivery-api#headers) header?

**Reply by bjarnef** [reply]
Ahh yes, that works :)
Thanks.

**Reply by jdpnielsen** [reply]
Be aware of a potential issue, if you are fetching content with language variations with fetch caching enabled: https://github.com/umbraco/Umbraco-CMS/issues/17034
